endpoints/openrtb2/video_auction_test.go

```diff
@@ -478,6 +478,16 @@ func TestVideoBuildVideoResponsePodErrors(t *testing.T) {
 	assert.Equal(t, int64(333), bidRespVideo.AdPods[2].PodId, "AdPods should contain error element at index 2")
 }
 
+func TestVideoBuildVideoResponseNoBids(t *testing.T) {
+	openRtbBidResp := openrtb.BidResponse{}
+	podErrors := make([]PodError, 0, 2)
```

Comment:
Any reason to initialize `podErrors` with a capacity of 2 here? 

Also, a minor suggestion (not a hard request): the next two lines can be potentially combined into: `openRtbBidResp.SeatBid = make([]openrtb.SeatBid, 0)` since `seatBids` is not used anywhere else.

Reply:
That was copy and paste, sorry. Fixed.